twint/run.py

```diff
@@ -69,7 +70,7 @@ def get_resume(self, resumeFile):
                     self.feed, self.init = feed.Json(response)
                 break
             except TimeoutError as e:
-                if self.config.Proxy_host.lower() == "tor":
+                if self.config.Proxy_host != None and self.config.Proxy_host.lower() == "tor":
```

Comment:
I'd just keep `self.config.Proxy_host.lower() == "tor"`, this condition is a sub-set of `self.config.Proxy_host != None`

Reply:
But then you will be calling the method lower into None,  that would raise the exception: None type has no method lower.  Other alternative is to do simply "if self.config.Proxy_host and ..." since None evaluates to false.

Reply:
So maybe we can change the default value (stored in `config.py`) to `config.Proxy_host = ""`?

Reply:
That should work!